<ui version="4.0" >
 <class>SceneEditor</class>
 <widget class="QDialog" name="SceneEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Scene editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QTabWidget" name="m_tab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="General" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QLabel" name="m_nameLabel" >
         <property name="text" >
          <string>Scene name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="m_nameEdit" >
         <property name="toolTip" >
          <string>Name of this scene</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QTreeWidget" name="m_tree" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Fixtures used in this scene</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="2" >
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QToolButton" name="m_addFixtureButton" >
           <property name="toolTip" >
            <string>Add a new fixture to this scene</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="main.qrc" >
             <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_removeFixtureButton" >
           <property name="toolTip" >
            <string>Remove the selected fixture(s) from this scene</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="main.qrc" >
             <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SceneEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>457</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel" >
     <x>158</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SceneEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>545</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel" >
     <x>308</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotEnableChannelsClicked()</slot>
  <slot>slotDisableChannelsClicked()</slot>
  <slot>slotUpdateValuesChecked(bool)</slot>
 </slots>
</ui>
